web/viewer.css

```diff
@@ -1452,8 +1452,8 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
 }
 
 .fileInput {
-  background: white;
-  color: black;
```

Comment:
Huh, why are these colours even set when the element has `visibility: hidden;`!?

Obviously *not* something that should to be fixed here, but I cannot help thinking that the CSS files might generally benefit from some clean-up since they probably contain a fair amount of rules only relevant for now unsupported browsers.

Reply:
That's... a very good question. I can only imagine that the visibility is toggled/used to be toggled using JavaScript. It probably cannot hurt to go over the CSS in a follow-up as an incremental improvement.